property.ts

```diff
@@ -151,7 +168,11 @@ export const checkProperties = (
           fc
             .record({
               functionArgsArb: fc.tuple(
-                ...functionToArbitrary(r.selectedTestFunction, simnetAddresses)
+                ...functionToArbitrary(
+                  r.selectedTestFunction,
+                  simnetAddresses,
+                  simnet
```

Comment:
Same comment about passing something more specific from the simnet here instead of the whole instance.